src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,9 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String model;
+    private int idNumber;
```

Comment:
Maybe you should initialise those fields through constructor with parameters? :)

Reply:
thanks, fix